engine/src/main/java/com/redhat/cloud/notifications/models/EmailAggregationKey.java

```diff
@@ -8,14 +8,21 @@ public class EmailAggregationKey {
     @NotNull
     private String accountId;
 
+    private String orgId;
+
     @NotNull
     private String bundle;
 
     @NotNull
     private String application;
 
-    public EmailAggregationKey(String accountId, String bundle, String application) {
+    // TODO why the heck do I need this constructor now?
+    public EmailAggregationKey() {
```

Comment:
@gwenneg do you have an idea why I have to add this constructor here? Before it was not needed and right now it is after I addd the orgId field. I don't recognize the reason why this is needed suddenly.

Reply:
Did you add it because of Hibernate?

Reply:
No, because of Kafka. I got deserialization exceptions after adding the orgId and I don't see why